src/care/ProfilePlugin.cpp

```diff
@@ -0,0 +1,86 @@
+#include "care/ProfilePlugin.h"
+
+#include "chai/ArrayManager.hpp"
+
+#include "chai/ExecutionSpaces.hpp"
+
+#include "care/CHAICallback.h"
+
+#include "care/RAJAPlugin.h"
+
+/* CUDA profiling macros */
+#if defined(__CUDACC__) && CARE_HAVE_NVTOOLSEXT
+#include "nvToolsExt.h"
+#endif
+
+// Std library headers
+#if defined(CARE_DEBUG) && !defined(_WIN32)
+#include <execinfo.h>
+#endif // defined(CARE_DEBUG) && !defined(_WIN32)
+
+#include <unordered_set>
+
+using namespace care;
```

Comment:
Technically not wrong, but doesn't match the rest of CARE stylistically. We don't really do "using namespace" anywhere.